src/tpm2_wrap.c

```diff
@@ -2605,6 +2661,27 @@ int wolfTPM2_NVWriteAuth(WOLFTPM2_DEV* dev, WOLFTPM2_NV* nv,
         wolfTPM2_SetAuthHandle(dev, 0, &nv->handle);
     }
 
+    XMEMSET((byte*)&inPublic, 0, sizeof(inPublic));
```

Comment:
Why not use `wolfTPM2_NVReadPublic`?

Reply:
Wow. I've forgot we have wolfTPM2 wrapper for that. Changed the code to use it - Awesome :)